### PR TITLE
Added school placements endpoints to statuscake monitoring

### PR DESCRIPTION
**Pull request by VenkataChalla554**
## Context

Add school placements endpoint's to statuscake monitoring.

## Changes proposed in this pull request

Endpoints added to staging, sanbox and production

## Guidance to review

 Endpoints added to staging, sanbox and production.

Terraform Plan  Sanbox  :  Adding resources related statuscake and monitoring alerts
Terraform Plan  Staging  :  Adding resources related statuscake and monitoring alerts
Terraform Plan  Production  : Adding resources related statuscake.

Terraform Plan  Production 👍 
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

    module.statuscake[0].statuscake_ssl_check.main["https://manage-school-placements.education.gov.uk"] will be created
  + resource "statuscake_ssl_check" "main" {
      + check_interval   = 3600
      + contact_groups   = [
          + "282453",
          + "311842",
        ]
      + follow_redirects = false
      + id               = (known after apply)
      + paused           = false

      + alert_config {
          + alert_at    = [
              + 3,
              + 7,
              + 30,
            ]
          + on_broken   = true
          + on_expiry   = true
          + on_mixed    = true
          + on_reminder = true
        }

      + monitored_resource {
          + address = "https://manage-school-placements.education.gov.uk"
        }
    }

   module.statuscake[0].statuscake_uptime_check.main["https://manage-school-placements.education.gov.uk/healthcheck/all"] will be created
  + resource "statuscake_uptime_check" "main" {
      + check_interval = 30
      + confirmation   = 2
      + contact_groups = [
          + "282453",
          + "311842",
        ]
      + id             = (known after apply)
      + locations      = (known after apply)
      + name           = "https://manage-school-placements.education.gov.uk/healthcheck/all"
      + paused         = false
      + regions        = [
          + "london",
          + "dublin",
        ]
      + trigger_rate   = 0

      + http_check {
          + enable_cookies   = false
          + follow_redirects = true
          + request_method   = "HTTP"
          + status_codes     = [
              + "204",
              + "205",
              + "206",
              + "303",
              + "400",
              + "401",
              + "403",
              + "404",
              + "405",
              + "406",
              + "408",
              + "410",
              + "413",
              + "429",
              + "444",
              + "494",
              + "495",
              + "496",
              + "499",
              + "500",
              + "501",
              + "502",
              + "503",
              + "504",
              + "505",
              + "506",
              + "507",
              + "508",
              + "509",
              + "510",
              + "511",
              + "520",
              + "521",
              + "522",
              + "523",
              + "524",
              + "598",
              + "599",
            ]
          + timeout          = 40
          + validate_ssl     = true
        }

      + monitored_resource {
          + address = "https://manage-school-placements.education.gov.uk/healthcheck/all"
        }
    }

Plan: 2 to add, 0 to change, 0 to destroy.

 

## Link to Trello card

 https://trello.com/c/YVZ1tzqN/1869-school-placements-statuscake-monitoring

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by VenkataChalla554** [comment]
[School placements statuscake monitoring](https://trello.com/c/YVZ1tzqN/1869-school-placements-statuscake-monitoring)

File: terraform/application/config/sandbox.tfvars.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
     "cluster": "production",
     "namespace": "bat-production",
+    "enable_monitoring" : true,
     "statuscake_alerts": {
-        "website_url": [ "https://sandbox.claim-funding-for-mentor-training.education.gov.uk/healthcheck/all" ],
+        "website_url": [ "https://sandbox.claim-funding-for-mentor-training.education.gov.uk/healthcheck/all","https://manage-school-placements.education.gov.uk/healthcheck/all" ],
```
Comment (RMcVelia) [inline terraform/application/config/sandbox.tfvars.json:6]: sandbox.manage...

File: terraform/application/config/sandbox.tfvars.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
         "contact_groups": [311842]
     }
 }
```
Comment (RMcVelia) [inline terraform/application/config/sandbox.tfvars.json:9]: Currently sandbox and staging do not work as enable_monitoring: true is not set. Underlying monitor group resources will also need to be created.

Comment (VenkataChalla554) [reply terraform/application/config/sandbox.tfvars.json:9]: "enable_monitoring" : true added tfvars

File: Makefile [hunk 3]
```diff
@@ -169,3 +169,9 @@ get-cluster-credentials: set-azure-account
 bin/konduit.sh:
 	curl -s https://raw.githubusercontent.com/DFE-Digital/teacher-services-cloud/main/scripts/konduit.sh -o bin/konduit.sh \
 		&& chmod +x bin/konduit.sh
+
+action-group-resources: set-azure-account # make production action-group-resources ACTION_GROUP_EMAIL=notificationemail@domain.com. Must be run before setting enable_monitoring=true for each subscription
+	$(if $(ACTION_GROUP_EMAIL), , $(error Please specify a notification email for the action group))
+	echo ${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-mn-rg
+	az group create -l uksouth -g ${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-mn-rg --tags "Product=Database of Qualified Teachers" "Environment=${DEPLOY_ENV}" "Service Offering=Database of Qualified Teachers"
```
Comment (RMcVelia) [inline Makefile:176]: tags are incorrect

Comment (VenkataChalla554) [reply Makefile]: Fixed this

**Review by RMcVelia — APPROVED** [review]

**Pull request merged**